packages/ui-components/src/components/windowedlist.ts

```diff
@@ -883,6 +944,7 @@ export class WindowedList<
     this.viewModel.height =
       msg.height >= 0 ? msg.height : this.node.getBoundingClientRect().height;
     super.onResize(msg);
+    this._updater.invoke();
```

Comment:
#15357 also invokes update but only if height has changed (as otherwise there is nothing to update I believe) but also adds a test and can be backported as it is self-contained. Would you be interested in reviewing it? I can add the throttler patterns as in this PR over in #15357.

Reply:
Good call, thank you @krassowski!